Clarify sorting for time zone IDs (#27890)
mdn/content: files/en-us/web/javascript/reference/global_objects/intl/supportedvaluesof/index.md

The **`Intl.supportedValuesOf()`** static method returns an array containing the supported calendar, collation, currency, numbering systems, or unit values supported by the implementation.

> Duplicates are omitted and the array is sorted in the same order as using {{jsxref("Array/sort", "Array.prototype.sort()")}} with an `undefined` compare function.
> All-uppercase or all-lowercase arrays will be sorted in ascending alphabetic order.
> However, for keys like `"timeZone"` that return an array with mixed-case strings, results will be sorted with uppercase ahead of lowercase, then alphabetically.

Reviewer: ```suggestion
Duplicates are omitted and the array is sorted in ascending lexicographical order (or more precisely, using {{jsxref("Array/sort", "Array.prototype.sort()")}} with an `undefined` compare function).
```

We usually just say this to mean the default `sort()` behavior.

Author: Works for me. Suggestion applied.
